carmaelement_mod.F90

```diff
@@ -38,7 +38,7 @@ module CARMAELEMENT_mod
   !! @see CARMA_AddGas
   !! @see CARMAELEMENT_Destroy
  subroutine CARMAELEMENT_Create(carma, ielement, igroup, name, rho, itype, icomposition, rc, &
-              shortname, isolute, rhobin, arat)
+              shortname, isolute, rhobin, arat, kappaElement)
```

Comment:
This is part of the Element object, so the kappa is by defintion the kappa of the element. Similar to other fields, should just call the parameter kappa.

Reply:
okay

Reply:
@fvitt  will test this